doi-requests-commons/src/main/java/no/unit/nva/doi/requests/model/AbstractDoiRequest.java

```diff
@@ -1,38 +0,0 @@
-package no.unit.nva.doi.requests.model;
```

Comment:
Deleted because it was used by only ApiUpdateDoiRequest which actually does not expect a pubication identifier in its body. The API endpoint expects the pubication identifier as a path parameter